app/presenters/setting_presenter.rb

```diff
@@ -50,6 +50,10 @@ def readonly?
     SETTINGS.key?(name.to_sym)
   end
 
+  def value
+    SETTINGS.fetch(name.to_sym) { super }
```

Comment:
where does `super` go in this case? this class has no parent?

Reply:
There's `attribute :value` which probably does some magic with `ActiveModel`, but it's a good point.

Reply:
it includes ActiveModel::Attributes which defines the super method.
Do you think it would be more clear to call the attribute fetch ?

Reply:
Not crucial but i think that might be a bit clearer to read
```suggestion
    SETTINGS.fetch(name.to_sym, attribute(:value))
```

Reply:
opened https://github.com/theforeman/foreman/pull/8883 to propose this change, feel free to close it if you disagree